.github/workflows/all-pr-tests.yml

```diff
@@ -45,3 +45,27 @@ jobs:
           php-version: '${{ matrix.php }}'
           wordpress-version: '${{ matrix.wordpress }}'
           skip-wordpress-install: 'true'
+  # This required job ensures that all PR checks have passed before merging.
+  all-pr-checks-passed:
+    name: All PR checks passed
+    needs:
+      - pr-tests
+    runs-on: ubuntu-latest
+    if: always()
+    permissions:
+      pull-requests: read
+      statuses: write
+      contents: read
```

Comment:
Never seen this before. What's the purpose?

Reply:
That part didn't need to come along :)